[Bug]: Fail to compile d499ffc

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: d499ffc
- Deployment mode(standalone or cluster): both
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

When compiling Milvus, got following error:
```
go: [github.com/soothing-rain/gorocksdb@v0.0.0-20220113075731-e68e68ed4c62](http://github.com/soothing-rain/gorocksdb@v0.0.0-20220113075731-e68e68ed4c62): invalid version: unknown revision e68e68ed4c62
Makefile:127: recipe for target 'build-cpp' failed
```


### Expected Behavior

Compilation succeeds

### Steps To Reproduce
run
```markdown
make  milvus
```


### Anything else?

_No response_

--- a/go.mod
+++ b/go.mod
@@ -30,7 +30,6 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/shirou/gopsutil v3.21.8+incompatible
-	github.com/soothing-rain/gorocksdb v0.0.0-20220113075731-e68e68ed4c62 // indirect
 	github.com/spaolacci/murmur3 v1.1.0
 	github.com/spf13/cast v1.3.1
 	github.com/spf13/viper v1.8.1

